polytracker/src/passes/polytracker_pass.cpp

```diff
@@ -543,82 +572,105 @@ bool PolytrackerPass::runOnModule(llvm::Module &mod) {
     function_index = (file_id << 24) | function_index;
   }
 
-  std::vector<llvm::Function *> functions;
-  for (auto &func : mod) {
-    // Ignore if its in our ignore list
-    if (func.hasName()) {
-      std::string fname = func.getName().str();
-      if (ignore_funcs.find(fname) != ignore_funcs.end()) {
-        continue;
+  if (no_control_flow_tracking) {
+    std::cout << "Omitting PolyTracker control flow instrumentation."
+              << std::endl;
+    // We still want to visit comparison functions because they are useful
```

Comment:
minor, for clarity: comparison instructions?

Reply:
Good catch!
```suggestion
    // We still want to visit comparison instructions because they are useful
```